src/components/WeekDayPicker/index.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import valuePropValidation from './helpers/valuePropValidation';
+import sortWeekDays from './helpers/sortWeekDays';
+import { useUniqueIdentifier } from '../../libs/hooks';
+import RenderIf from '../RenderIf';
+import WeekDayItems from './weekDayItems';
+import Label from '../Input/label';
+import StyledTextError from '../Input/styled/errorText';
+import { StyledFieldset, StyledHelpText } from './styled';
+
+/**
+ * A WeekDayPicker allows to select the days of the week
+ * @category Form
+ */
+function WeekDayPicker(props) {
+    const {
+        id,
+        name,
+        value,
+        label,
+        bottomHelpText,
+        availableDates,
+        disabled,
+        required,
+        readOnly,
+        multiple,
+        error,
+        onChange,
+        className,
+        style,
+    } = props;
+
+    const defaultFieldsetName = useUniqueIdentifier('week-day-items');
+    const fieldsetName = name || defaultFieldsetName;
+
+    const prepareReturnValue = (weekDayValue, isChecked) => {
+        if (multiple) {
+            if (isChecked && !value.includes(weekDayValue)) {
+                return sortWeekDays([...value, weekDayValue]);
+            }
+
+            return sortWeekDays(value.filter(day => day !== weekDayValue));
+        }
+
+        return weekDayValue;
+    };
+
+    const handleOnChange = e => {
+        const weekDayValue = e.currentTarget.value;
+        const isChecked = e.currentTarget.checked;
+
+        return onChange(prepareReturnValue(weekDayValue, isChecked));
+    };
+
+    return (
+        <StyledFieldset className={className} style={style} id={id}>
+            <Label label={label} hideLabel={!label} required={required} readOnly={readOnly} />
```

Comment:
sorry for this, let's move this back as before since this need to be a legend element and Label components uses a label tag